patterns/ecs-v1/linux-syslog

```diff
@@ -1,16 +1,16 @@
 SYSLOG5424PRINTASCII [!-~]+
 
-SYSLOGBASE2 (?:%{SYSLOGTIMESTAMP:timestamp}|%{TIMESTAMP_ISO8601:timestamp8601}) (?:%{SYSLOGFACILITY} )?%{SYSLOGHOST:logsource}+(?: %{SYSLOGPROG}:|)
-SYSLOGPAMSESSION %{SYSLOGBASE} (?=%{GREEDYDATA:message})%{WORD:pam_module}\(%{DATA:pam_caller}\): session %{WORD:pam_session_state} for user %{USERNAME:username}(?: by %{GREEDYDATA:pam_by})?
+SYSLOGBASE2 (?:%{SYSLOGTIMESTAMP:timestamp}|%{TIMESTAMP_ISO8601:timestamp})(?: %{SYSLOGFACILITY})?(?: %{SYSLOGHOST:[host][hostname]})?(?: %{SYSLOGPROG}:)?
```

Comment:
:+1: to capturing to single field `timestamp`.

Propbably need to follow up on the semantics of `[host][hostname]` vs `[host][name]` here too.

Reply:
filebeat is filling [`host.hostname`](https://github.com/elastic/beats/blob/v7.9.2/filebeat/module/system/syslog/test/suse-syslog.log-expected.json#L8) (as opposed to `host.name`) so it made sense to keep that ... at least as a start